main.py

```diff
@@ -34,21 +38,21 @@ def display(user_prompt, content):
         store_to_gsheet(content, user_prompt=user_prompt, error=True, **st.secrets.gspread_credentials)
     else:
         output = dict_to_mermaid(output)
-        store_to_gsheet(content, user_prompt=user_prompt, error=False, **st.secrets.gspread_credentials)
+        store_to_gsheet(content, user_prompt = user_prompt, error = True, **st.secrets.gspread_credentials)        # passing google sheet credentials
```

Comment:
What is the reasoning behind changing the boolean in function?

Reply:
I was just checking for errors, but there's no actual necessity to set it to 'true.' As long as the application runs successfully, there's no need to worry about it. Although, I should probably change it back to 'false,' I guess.